apps/angular/src/core/dtos/anime-management-params.dto.ts

```diff
@@ -0,0 +1,18 @@
+/** Paginated anime data dto. */
+export type AnimeManagementParamsDto = {
+
+	/** Limit. */
+	readonly limit: string;
+
+	/** Offset. */
+	readonly offset: string;
```

Comment:
Why these properties have 'string' type? I guess `number` should be used here instead:
```suggestion
	/** Limit. */
	readonly limit: number;

	/** Offset. */
	readonly offset: number;
```